### PR TITLE
Adds an initial implementation of an update the state of the commands of a task.

### PR DESCRIPTION
**Pull request by emanueljoivo**
Don't merge this right now. Arrebol needs to offer the commands states of a task before.

### CODE REVIEW

File: src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java [hunk 1]
```diff
@@ -43,11 +45,34 @@ public JDFJob synchronizeJob(JDFJob job) {
     }
 
     private void updateJob(JDFJob job, ArrebolJob arrebolJob) {
+        updateCommandsState(job.getTaskList(), arrebolJob.getTasks());
         updateTasksState(job.getTaskList(), arrebolJob.getTasks());
         LOGGER.info("Updated tasks state from job [" + job.getId() + "].");
         updateJobState(job, arrebolJob.getJobState());
     }
 
+    private void updateCommandsState(Map<String, Task> iguassuTasks, List<ArrebolTask> arrebolTasks) {
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java]: Please, also update the command exit code, the next feature that we intend to address is the sync of all commands attributes between Iguassu and Arrebol, and doing it now will avoid the need to patch new implementations to Iguassu.

File: src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java [hunk 2]
```diff
@@ -82,14 +107,30 @@ private boolean tasksFinished(JDFJob job) {
         return job.getTasks().size() == tasksFinished;
     }
 
+    private CommandState getCommandState(ArrebolCommandState arrebolCommandState) {
+        switch (arrebolCommandState) {
+            case RUNNING:
+            case UNSTARTED:
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java]: Why UNSTARTED should be mapped to RUNNING state?

File: bin/shutdown.pid [hunk 3]
```diff
@@ -0,0 +1 @@
+14874
```
Comment (paulofelipefeitosa) [inline bin/shutdown.pid]: Please remove this file.

File: src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java (hunk 2 above)
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java]: Maybe PENDING or literally UNSTARTED will be better.

File: bin/shutdown.pid (hunk 3 above)
Comment (emanueljoivo) [reply bin/shutdown.pid]: This was solved in df5aa7d3bda8d19bf734ebaf2e7dc505a5f60ac3.

**Pull request merged**